panther_driver/src/driver_node.py

```diff
@@ -317,49 +329,85 @@ def driverNode():
                 rospy.logerr("Error Calculating and publishing bat data")
 
 
-            # inverse kinematics
+            # query position
+            try:
+                wheel_pos[0] = front_controller.sdo['Qry_ABCNTR'][2].raw
+            except:
+                rospy.logwarn("Error reading front left controller sdo")
+            try:
+                wheel_pos[1] = front_controller.sdo['Qry_ABCNTR'][1].raw
+            except:
+                rospy.logwarn("Error reading front right controller sdo")
+            try:
+                wheel_pos[2] = rear_controller.sdo['Qry_ABCNTR'][2].raw
+            except:
+                rospy.logwarn("Error reading rear left controller sdo")
+            try:  
+                wheel_pos[3] = rear_controller.sdo['Qry_ABCNTR'][1].raw
+            except:
+                rospy.logwarn("Error reading rear right controller sdo")
```

Comment:
I think that we may want better error handling, but it will probably better to do in grand refactor, for now just leaving a comment.